app/models/solr_document.rb

```diff
@@ -40,10 +40,6 @@ def logical_order
       end
   end
 
-  def language_display
```

Comment:
nope just for display with the individual record on the search results. The language gets mapped at index time for the facet: https://github.com/pulibrary/plum/blob/master/app/indexers/work_indexer.rb#L27-L29